File: Cs-Connect-java/app/src/main/java/com/csestateconnect/www/csconnect/models/search/SearchResult.java
package com.csestateconnect.www.csconnect.models.search;


public class SearchResult {

    private Integer id;
    private String name;
    private String type;
    private String location;

    public SearchResult(String mName, String mType){
        this.name = mName;
        this.type = mType;
    }

    public SearchResult(Integer mId, String mName, String mType, String mLocation){
        this.id = mId;
        this.name = mName;
        this.type = mType;
        this.location = mLocation;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public String getLocation() {
        return location;
    }
}

File: CsConnect/app/src/main/java/com/csestateconnect/db/data/SearchResult.kt
package com.csestateconnect.db.data

class SearchResult {

    private var id: Int? = null
    private var name: String? = null
    private var type: String? = null
    private var location: String? = null

    constructor(mName: String, mType: String) {
        this.name = mName
        this.type = mType
    }

    constructor(mId: Int?, mName: String, mType: String, mLocation: String) {
        this.id = mId
        this.name = mName
        this.type = mType
        this.location = mLocation
    }

    fun getId(): Int? {
        return id
    }

    fun getName(): String? {
        return name
    }

    fun getType(): String? {
        return type
    }

    fun getLocation(): String {
        return location!!
    }
}